tests/maas-install.sh

```diff
@@ -18,10 +18,29 @@
 
 set -evu
 
+# sets the desired maas_release to test
+MAAS_RELEASE=master
```

Comment:
will this ever need to be configurable?

Reply:
Guess it depends on if we stabilize on a version or not... I could do what rpc-o is doing and grab the latest version and test that, but currently the latest version isn't installing yaml which is needed to pass when master does.